src/lando/urls.py

```diff
@@ -52,6 +52,11 @@
     path("D<int:revision_id>/", revisions.Revision.as_view(), name="revisions-page"),
     path("manage_api_key/", user_settings.manage_api_key, name="user-settings"),
     path("uplift/", revisions.Uplift.as_view(), name="uplift-page"),
+    path(
+        "uplift-assessment/",
```

Comment:
Should we namespace a bit more?
```suggestion
        "uplift/assessment/",
```